README.md

```diff
@@ -1,10 +1,7 @@
-## IdP customizations for UW
+# IdP customizations for UW
 
 These files are customization configuration for the UW Shib IdP.
 
-Presently they are for reference only.  Not meant to be installed.
-
-Jim Fox
-fox@uw.edu
-
+Our idp servers are at:
 
+- idp1[1-4].s.uw.edu
```

Comment:
This needs changed to idp1[1-6]. We have six prod servers.

Reply:
Fixed, thanks!